tools/runners/run-all.js

```diff
@@ -214,7 +215,7 @@ class Runner {
       runLog.log("");
       if (process.env.UNIX_SOCKET_PATH) {
         runLog.log(
-          `App running; UNIX domain socket: ${process.env.UNIX_SOCKET_PATH}`,
+            `App running; UNIX domain socket: ${process.env.UNIX_SOCKET_PATH} RootUrl: ${self.rootUrl}`,
```

Comment:
One extra indentation level creeped in. Also, perhaps we want to print out "App running at: ", exactly as the non-UNIX-domain-socket version below? It's a bit more readable, since root URL is what user likely cares about (that's a clickable link), and UNIX domain socket path is a rather arcane internal detail. We can print it in addition to the regular line (could be on a separate line).

Reply:
Fixed